lib/api.js

```diff
@@ -48,11 +61,12 @@ function Api (plugins, defaultConfig) {
     const opts = merge(merge({}, defaultConfig), inputOpts)
     // change event emitter to something with more rigorous security?
     let api = new EventEmitter()
-    create.plugins.forEach((plug) => {
+    for (const plug of create.plugins) {
+      const subOpts = pluckOpts(opts, plug)
```

Comment:
Rename to be config (not opts)
Recommend pluckOpts becomes buildSubConfig or buildPlugGonfig

Reply:
Reason. Its config not opts... why have two name for the same thing